src/clients/UBAClient/UBAClientBase.ts

```diff
@@ -358,6 +360,7 @@ export abstract class BaseUBAClient {
     if (state) {
       this.bundleStates = state;
     }
+    this.isUpdated = true;
```

Comment:
Should this be `_isUpdated`? 

Also, I'm a bit confused how this extension works. The other clients IIUC all set `isUpdated = true` at the end of their `update()` calls. Does this mean we need to change those calls to set `_isUpdated = true` instead?

Reply:
https://github.com/across-protocol/sdk-v2/pull/277#discussion_r1258332373

The `get` `set` definitions in TS allow us to enforce/dynamically generate values from accessing the variables directly.

We can use `isUpdated` and `isUpdated = ...` anywhere in the code now and TS will implicitly route the logic back to the functions created in the base abstract client.